tests/e2e/visual/reports/report-filter.wdio-spec.js

```diff
@@ -0,0 +1,50 @@
+const utils = require('@utils');
+const commonElements = require('@page-objects/default/common/common.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const { resizeWindowForScreenshots, generateScreenshot } = require('@utils/screenshots');
+const { reports, contact } = require('./data/generateReportData');
+
+describe('Report Filter functionality test', () => {
+  const docs = [contact, ...reports];
+  const savedUuids = [];
+  const reportUuids = [];
```

Comment:
Neither variable is used anywhere else in the file besides being populated in the before hook.